README.md

```diff
@@ -1,3 +1,5 @@
+[![license](https://img.shields.io/github/license/mashape/apistatus.svg)](LICENSE.txt) [![CircleCI](https://img.shields.io/circleci/project/github/RedSparr0w/node-csgo-parser.svg)](https://circleci.com/gh/vintasoftware/django-react-boilerplate)
```

Comment:
As @aericson said, we already have CircleCI badge and for the license, you'll just need to make it all uppercase(LICENSE.TXT instead of LICENSE.txt) since that's the correct filename. Remember it's case sensitive.